src/content/docs/cachyos_repositories/what_are_the_cachyos_repo.md

```diff
@@ -1,31 +1,45 @@
 ---
 title: CachyOS Repository General Information
-description: Benchmarks and information about the repository
+description: Benchmarks and information about the repositories
 ---
 
-# Why does CachyOS utilize its own repositories?
+Our aim to deliver a performance-optimized distribution necessitates recompiling essential Arch Linux packages for generic x86-64,
+x86-64-v3, x86-64-v4, and Zen4 architectures.
 
-## Performance and optimizations
-We want to provide a performance-optimized distribution and this requires performance-optimized packages. Currently we are recompiling the Arch Linux repository core and extra with the generic x86-64-v3 and x86-64-v4 instruction set.
+- x86-64 (Most CPUs) - While lacking AVX2 support found in x86-64-v3, this repository offers optimized packages. LTO and PGO techniques have boosted performance in several packages.
+- x86-64-v3- (Intel Haswell and up, AMD Ryzen 1st gen and up) - 5%-20% performance uplift compared to x86-64.
+- x86-64-v4 (Intel Rocket Lake, Zen 4, Zen 5) - Delivers substantial performance gains through AVX512 support, depending on the workload.
+- Zen 4 - In addition to the x86-64-v4 instruction set, the following are added:
 
-- x86-64-v4 - Enabling AVX512 can significantly improve application performance (Depending on the application/workload)
-- x86-64-v3 - *5%-20% performance uplift*
+```bash
```

Comment:
This kinda fucks up the syntax highlighting. It doesn't look that bad tho so I guess we can leave it

Reply:
Its to follow Markdown's md040 guidelines.

Reply:
Although it can be changed for another language in order to have syntax highlighting

Reply:
I found a solution, pushing...